jaxrs-api/pom.xml

```diff
@@ -680,9 +680,9 @@
         <maven.javadoc.plugin.version>3.0.0</maven.javadoc.plugin.version>
 
         <api.package>javax.ws.rs</api.package>
-        <last.final.spec.version>2.1</last.final.spec.version>
+        <last.final.spec.version>2.0</last.final.spec.version>
```

Comment:
I agree that these properties are really confusing. The `final` profile sets the effective spec version (used for the manifest any maybe in other places) to the value of `${next.final.spec.version}`. So setting it to 2.1 in the `EE4J_8` branch makes sense to me.

https://github.com/eclipse-ee4j/jaxrs-api/blob/20223096b812df674c54cc44d50a7759c56309c5/jaxrs-api/pom.xml#L78-L90

Running the build with the `final` profile looks good now:

```
$ mvn -Pfinal clean install
[...]
[INFO] -- spec properties --
[INFO] spec.implementation.version = 2.1.2
[INFO] spec.specification.version = 2.1
[INFO] spec.bundle.symbolic-name = javax.ws.rs-api
[INFO] spec.bundle.spec.version = 2.1
[INFO] spec.extension.name = javax.ws.rs
[INFO] spec.bundle.version = 2.1.2
```

Without the `final` profile it looks incorrect:

```
$ mvn clean install
[...]
[INFO] --- spec-version-maven-plugin:1.5:set-spec-properties (default) @ jakarta.ws.rs-api ---
[INFO] 
[INFO] -- spec properties --
[INFO] spec.implementation.version = 2.1.2
[INFO] spec.specification.version = 2.0.99.01
[INFO] spec.bundle.symbolic-name = javax.ws.rs-api
[INFO] spec.bundle.spec.version = 2.0.99.b01
[INFO] spec.extension.name = javax.ws.rs
[INFO] spec.bundle.version = 2.0.99.b01
```

However, as JAX-RS 2.1 actually IS final, it doesn't make sense any more to run without the `final` profile. So we could actually make it the default.

Reply:
This is exactly why I want to get rid of that plugin. This is ancient Oracle-vodoo. We should completely replace the magic (including that profile) by manually maintained manifests.

Reply:
With "ancient Oracle-vodoo" I meant the fact that Oracle wrote their own, mostly undocumented, plugin within the "GlassFish" project, instead of participating to well-documented, cross-vendor plugin that people outside of that company knows and understands.

Reply:
I guess that's because the plugin's purpose is mostly to enforce the some [Glassfish-specific conventions](https://javaee.github.io/glassfish/wiki-archive/Maven%20Versioning%20Rules.html). However, I agree that some kind of documentation would be very useful.

Reply:
2.1.2 has already been released. This would need to be 2.1.3.

Reply:
Yes, sure. The pom version wasn't updated yet, so the output still shows the wrong version number.